cedar-drt/fuzz/src/schemas.rs

```diff
@@ -85,14 +107,11 @@ fn action_type_equivalence(name: &str, lhs: ActionType, rhs: ActionType) -> Resu
     }
 }
 
-fn empty_target(spec: &ApplySpec) -> bool {
-    spec.resource_types
-        .as_ref()
-        .map(|v| v.is_empty())
-        .unwrap_or(false)
-        || spec
-            .principal_types
-            .as_ref()
-            .map(|v| v.is_empty())
-            .unwrap_or(false)
+fn is_both_unspecified(spec: &ApplySpec) -> bool {
+    spec.resource_types.is_none() && spec.principal_types.is_none()
+}
+
+fn is_either_empty(spec: &ApplySpec) -> bool {
```

Comment:
Nit: Like above, change to `either_empty`.

Reply:
Done.